server/pres.go

```diff
@@ -295,6 +295,17 @@ func (t *Topic) presSubsOnlineDirect(what string) {
 	}
 }
 
+// Communicates "topic unaccessible (cluster rehashing or node connection lost)" event
+// to a list of topics promting the client to resubscribe to the topics.
+func (s *Session) presTermDirect(subs []string) {
+	log.Printf("sid '%s', uid '%s', terminating %s", s.sid, s.uid, subs)
+	msg := &ServerComMessage{Pres: &MsgServerPres{Topic: "me", What: "term"}}
+	for _, topic := range subs {
+		msg.Pres.Src = topic
```

Comment:
This is likely to generate wrong topic names for p2p topics.

Reply:
That's why I'm adding remSub.originalTopic to the array which gets passed to this function.
Let me double-check it.

Reply:
Just re-verified: it does generate short names {"pres":{"topic":"me","src":"usrj_PEbl0quUg","what":"term"}}

Reply:
Thanks!
What about new group topics, i.e. cases when the original topic name is like `new123`?

Reply:
Good catch! Changed the code to use expanded topic name.